packages/nftoupon/src/Connect.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
```

Comment:
when you try to rebase please make sure you rebase with appropriate changes.... 
This has to be in the component folder... Don't worry will fix it..